ibis/expr/operations.py

```diff
@@ -720,6 +720,17 @@ class Quantile(Reduction):
     output_type = rules.type_of_arg(1)
```

Comment:
Also I think the `number(name='quantile')` should be `double(name='quantile')` since we only allow doubles there.